src/main/java/com/jfrog/ide/common/utils/Constants.java

```diff
@@ -1,8 +1,12 @@
 package com.jfrog.ide.common.utils;
 
+import org.jfrog.build.client.ArtifactoryVersion;
+
 /**
  * @author yahavi
  */
 public class Constants {
+    public static final ArtifactoryVersion MINIMAL_ARTIFACTORY_VERSION_SUPPORTED = new ArtifactoryVersion("7.16.3");
```

Comment:
Since older Artifactoey versions are partially supported, let's conside renaming this constant or (if it makes sense), remove it altogether.
The same goes for the ```isArtifactoryVersionSupported``` method.

Reply:
Changed to RECOMMENDED_ARTIFACTORY_VERSION_SUPPORTED